README.md

```diff
@@ -1 +1,41 @@
-# node-modules-folder-lab
\ No newline at end of file
+# Importing Folders (Proxy for Modules)
+
+## Objectives
+
+1. Implement a folder as a proxy for modules
+1. Create files in a folder, files with exports
+1. Import files/modules from another file via folder "proxy"
+
+## Introduction
+
+In this lab, you'll learn how to proxy folders or in other words how to create an abstraction layer which will allow you to group files using the Node feature of importing folders.
```

Comment:
Reframe this - What is the "domain" of the lab, ie. what kind of files are they working with, what kind of project is this contributing to?